tgui/packages/tgui/interfaces/PacketSniffer.tsx

```diff
@@ -0,0 +1,48 @@
+/**
+ * @file
+ * @copyright 2025
+ * @author FlameArrow57 (https://github.com/FlameArrow57)
+ * @license ISC
+ */
+
+import { Button, LabeledList, Section } from 'tgui-core/components';
+
+import { useBackend } from '../backend';
+import { Window } from '../layouts';
+
+interface PacketInfo {
+  packet_stamps;
+  packet_data;
+  filter;
```

Comment:
This is a typescript file so these should have some kind of type annotations.

Reply:
Sure, can you clarify what you mean by type annocations though? if you are meaning stuff like "filter: string;", I did not see any of that in the other .tsx files

Reply:
You were looking at files that weren't fully converted yet, we were just doing it to get TGUI to modern standards